runing/schemas.py

```
@@ -20,3 +21,21 @@ class User(UserBase):
     class Config:
         from_attributes = True
 
+class Item(BaseModel):
+    name: str
+    description: str
+    price: int
+    is_active: bool = True
+
+    class Config:
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)